repos/rust-lang/compiler-team.toml

```diff
@@ -0,0 +1,14 @@
+org = "rust-lang"
+name = "compiler-team"
+description = "A home for compiler team planning documents, meeting minutes, and other such things."
+bots = []
```

Comment:
Does rustbot need to be added here?

Reply:
Is it used there? Just to make sure I understand it properly, does the existence of `triagebot.toml` in the root of the repo imply that the repo used rustbot?

Reply:
It is used (e.g., [here](https://github.com/rust-lang/compiler-team/issues/710#issuecomment-1875578907)). I believe the existence of that config does imply that it's used, but I was just going by issues where rustbot made an appearance.

Reply:
Ok, added it.